.github/workflows/testing.yml

```
@@ -95,7 +95,7 @@ jobs:
       run: python -m pip install --progress-bar off --upgrade tox
 
     - name: Build docs
-      run: tox -e build_docs_pins -- -q
+      run: tox -e build_docs_pins
```

```suggestion
      run: tox -e build_docs_pins -- -q
```
Making these quiet again for the merging